libcaf_core/src/detail/rfc3629.cpp

```diff
@@ -72,13 +69,19 @@ bool validate_rfc3629(const std::byte* first, const std::byte* last) {
         return false;
       continue;
     }
-    // 1111'0bxx: 4-byte sequence.
+    // 1111'0xxx: 4-byte sequence.
     if (head<5>(x) == 0b1111'0000_b) {
+      // Out of valid UTF range
```

Comment:
I had to read this comment a few times, at least to me it reads a bit ambiguous. Maybe this?

```suggestion
      // Check if code point is in the valid UTF range.
```